jekyll/_cci2/server/operator/managing-orbs.adoc

```diff
@@ -13,13 +13,13 @@ version:
 
 This section describes how to manage orbs for an installation of server v4.x. Server installations include their own local orb registry. All orbs referenced in configs refer to the orbs in the server orb registry. You are responsible for maintaining orbs. This includes copying orbs from the public registry, updating orbs that may have been previously copied, and registering your company's private orbs, if they exist.
 
-For information on orbs and related use cases, see the https://circleci.com/docs/2.0/orb-intro/#quick-start[Orb docs].
+For information on orbs and related use cases, see the https://circleci.com/docs/orb-intro/#quick-start[Orb docs].
```

Comment:
@rosieyohannan Would you happen to know where this should link to?

Reply:
How odd! I have a pr open to add a quickstart section to that page. Anyway, I think the top of the intro page is good here.

Reply:
Sorry, I'm unclear on this - this link is good as is?